internal/conn/storage/storage.go

```diff
@@ -174,9 +174,9 @@ func (c *Client) Upload(ctx context.Context, id string, b []byte) (*url.URL, err
 	}
 
 	if c.contExt == "" {
-		cName = fmt.Sprintf("%s-%s", contPrefix, c.now().UTC().Format(time.DateOnly))
+		cName = fmt.Sprintf("%s-%s-%d", contPrefix, c.now().UTC().Format(time.DateOnly), time.Now().Hour())
```

Comment:
approved but actually this could use a unit test

Reply:
Done